build/windows/download_zip_extract_exe.ps1

```diff
@@ -0,0 +1,19 @@
+param (
+    [string]$INTEGRATION="none",
+    [string]$ARCH="amd64",
+    [string]$TAG="v0.0.0",
+    [string]$REPO_FULL_NAME="none"
+)
+write-host "===> Creating dist folder"
+New-Item -ItemType directory -Path .\dist
+
+$VERSION=${TAG}.substring(1)
```

Comment:
There is no check that the tag is really formatted as vx.x.x

I run the apache pipeline with 0.0.1 as tag and then this script was trying to download version .0.1

Reply:
https://github.com/newrelic/nri-apache/runs/1689540462?check_suite_focus=true#step:4:30

Reply:
The 
```yaml
    tags:
      - 'v*'
```
on the prerelease workflow trigger should prevent this now.